cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -196,6 +218,34 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
     cy.get(asset_properties_widget_elements.saveButton).should('be.visible');
   });
 
+    // Change the order of properties and click on cancel button,Verify that the changes are not being saved.
+  it('TC_Asset_Properties_Widget_config_009', () => {
+    cy.get(moveBars3).dragTo(moveBars1);
+    cy.get(asset_properties_widget_elements.cancelButton).click();
+    cy.get(asset_properties_widget_elements.addWidgetButton).click();
+    cy.get(asset_properties_widget_elements.cardElement).eq(0).click();
+    cy.get(labelElement).eq(0).should('have.value', 'Name');
+  });
+
+  // Change the order of properties and click on save button,Verify if the order in which user has changed is properly getting displayed on widget cover.
+  it('TC_Asset_Properties_Widget_config_010', () => {
+    cy.selectAsset(assetName);
+    cy.wait(1000);
```

Comment:
provide information on why this static wait is needed. If this is to wait for some call to complete, use intercepts

Reply:
It is required to add wait so that drag will work as expected otherwise drag will not work